src/Raven.Server/Documents/ETL/Providers/OLAP/ParquetTransformedItems.cs

```diff
@@ -0,0 +1,445 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using Parquet;
+using Parquet.Data;
+using Raven.Client.Documents.Operations.ETL.OLAP;
+using Sparrow.Json;
+using Sparrow.Logging;
+
+namespace Raven.Server.Documents.ETL.Providers.OLAP
+{
+    public class ParquetTransformedItems : OlapTransformedItems
+    {
+        private bool[] _boolArr;
+        private string[] _strArr;
+        private double[] _doubleArr;
+        private long[] _longArr;
+        private DateTimeOffset[] _dtoArr;
+        private TimeSpan[] _tsArr;
+
+        public const string DefaultIdColumn = "_id";
+        public const string DefaultPartitionColumn = "_dt";
+        public const string LastModifiedColumn = "_lastModifiedTicks";
+
+        public ParquetTransformedItems(string name, string key, string idColumn, string partitionColumn) : base(OlapEtlFileFormat.Parquet)
+        {
+            CollectionName = name;
+            PartitionKey = key;
+            Groups = new List<RowGroup>();
+
+            _dataFields = new Dictionary<string, DataType>();
+
+            if (string.IsNullOrEmpty(partitionColumn))
+                partitionColumn = DefaultPartitionColumn;
+
+            DocumentIdColumn = string.IsNullOrEmpty(idColumn)
+                ? DefaultIdColumn
+                : idColumn;
+
+            Prefix = $"{CollectionName}/{partitionColumn}%3D{PartitionKey}";
```

Comment:
What is `%3D`? Why it needs to be provided this way? Please create a dedicated const with proper name for that